docs/reference/docs/index_.asciidoc

```diff
@@ -1,11 +1,124 @@
 [[docs-index_]]
 === Index API
+++++
+<titleabbrev>Index</titleabbrev>
+++++
 
 IMPORTANT: See <<removal-of-types>>.
 
-The index API adds or updates a JSON document in a specific index,
-making it searchable. The following example inserts the JSON document
-into the "twitter" index with an id of 1:
+Adds a JSON document to the specified index and makes
+it searchable. If the document already exists
+updates the document and increments its version.
+
+[[docs-index-api-request]]
+==== {api-request-title}
+
+`PUT /<index>/_doc/<_id>`
+
+`POST /<index>/_doc/`
+
+[[docs-index-api-path-params]]
+==== {api-path-parms-title}
+
+`<index>`::
+(Required, string) Name of the target index. By default, the index is created
+automatically if it doesn't exist. For more information, see <<index-creation>>.
+
+`_doc`::
```

Comment:
I don't think we should document `_doc` as it's no longer a changeable param. We can just remove this definition.

Reply:
Removed & addressed _doc vs _create in the detailed description section.